src/components/Label/Label.scss

```diff
@@ -211,5 +247,23 @@ $transitionTimingFunction: ease-in-out;
                 var(--yc-color-text-complementary)
             );
         }
+
+        &_plain {
+            @include themeState(
+                var(--yc-color-base-generic),
+                var(--yc-color-base-generic-hover),
+                var(--yc-color-text-complementary)
+            );
+
+            // hover on action button
+            &:not(#{$disabled}) #{$block}__icon_interactive {
+                --yc-button-background-color: transparent;
+                --yc-button-background-color-hover: var(--yc-color-text-secondary);
+
+                &:hover {
+                    color: var(--yc-color-text-inverted-primary);
+                }
+            }
```

Comment:
This code copy-pasted from `themeState` mixin with a tiny change, maybe there is a way to avoid it?